sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/README.md

```diff
@@ -0,0 +1,252 @@
+* An existing Key Vault. If you need to create a Key Vault, you can use the [Azure Cloud Shell](https://shell.azure.com/bash) to create one with this Azure CLI command:
+
+    `az keyvault create --resource-group <resource-group-name> --name <key-vault-name>`
+
+### Authenticate the client
+In order to interact with the Key Vault service, you'll need to create an instance of the [KeyVaultClient](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault) class. You need a vault url, client secret credentials (client id, client secret, tenant id) and a [resource url](https://vault.azure.net) to instantiate a client object.
+
+ #### Create/Get credentials
+Use the [Azure CLI](https://docs.microsoft.com/cli/azure) snippet below to create/get client secret credentials.
+
+ * Create a service principal and configure its access to Azure resources:
+
+     `az ad sp create-for-rbac -n <your-application-name> --skip-assignment`
+    > If you don't specify a password, one will be created for you.
+* Grant the above mentioned application authorization to perform secret and key operations on the keyvault:
+     `az keyvault set-policy --name <key-vault-name> --spn <your-service-principal-id> --secret-permissions <secret-permissions> --key-permissions <key-permissions>`
+
+#### Create Secret client
+Once you have the `client_id`, `client_secret` and `tenant_id` from above, you can create the [SecretClient](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/_client.py):
+```python
+    from azure.identity import AsyncClientSecretCredential
+    from azure.security.keyvault import SecretClient
+
+    credential = AsyncClientSecretCredential(client_id=client_id, secret=client_secret, tenant_id=tenant_id)
+
+    # Create a new secret client using a client secret credential
+    secret_client = SecretClient(vault_url=vault_url, credential=credential)
+```
+## Key concepts
+### Secret
+  A secret is the fundamental resource within Azure KeyVault. From a developer's perspective, Key Vault APIs accept and return secret values as strings. In addition to the secret data, the following attributes may be specified:
+* expires: Identifies the expiration time on or after which the secret data should not be retrieved.
+* not_before: Identifies the time after which the secret will be active.
+* enabled: Specifies whether the secret data can be retrieved.
+* created: Indicates when this version of the secret was created.(read-only)
+* updated: Indicates when this version of the secret was updated.(read-only)
+
+### Secret Client:
+The Secret client performs the interactions with the Azure Key Vault service for getting, setting, updating,deleting, and listing secrets and its versions. An asynchronous and synchronous, SecretClient, client exists in the SDK allowing for selection of a client based on an application's use case.
```

Comment:
Add at the end: Once you've initialized a SecretClient, you can interact with the primary resource types in Key Vault